site/en/docs/extensions/reference/dns/index.md

```diff
@@ -0,0 +1,43 @@
+---
+api: dns
+---
+
+## Manifest
+
+To use this API, you must declare the `"dns"` permission in the [manifest][manifest].
+
+```json
+{
+  "name": "My extension",
+  ...
+  "permissions": [
+    "dns"
+  ],
+  ...
+}
+```
+
+{% Aside %}
+This API is only available in [Chrome Dev](https://www.google.com/chrome/dev/).
```

Comment:
I would explicitly mention that there are not plans for it to move out of dev channel with the normal chrome release train.